src/pages/freelancers/index.tsx

```diff
@@ -353,40 +380,83 @@ const Freelancers = (): JSX.Element => {
-                      height={300}
-                      width={300}
-                      alt=""
-                    />
-                    <div className="dark-layer" />
-                  </div>
-                  <div className={styles.freelancerInfo}>
-                    <h3>{display_name}</h3>
-                    <h5>{title}</h5>
-                    <div className={styles.skills}>
-                      {skills?.slice(0, 3).map((skill, index) => (
-                        <p className={styles.skill} key={index}>
-                          {skill.name}
-                        </p>
-                      ))}
+            {freelancers?.length &&
+              paginatedFreelancers?.()
+                ?.currentFreelancers?.slice?.(0, 10)
+                ?.map?.(
+                  (
+                    { title, username, display_name, skills, profile_image },
+                    index
+                  ) => (
+                    <div className={styles.freelancer} key={index}>
+                      <div className={styles.freelancerImageContainer}>
+                        <Image
+                          src={
+                            profile_image ??
+                            require("@/assets/images/profile-image.png")
+                          }
+                          className={styles.freelancerProfilePic}
+                          height={300}
+                          width={300}
+                          alt=""
+                        />
+                        <div className="dark-layer" />
+                      </div>
+                      <div className={styles.freelancerInfo}>
+                        <h3>{display_name}</h3>
+                        <h5>{title}</h5>
+                        <div className={styles.skills}>
+                          {skills?.slice(0, 3).map((skill, index) => (
+                            <p className={styles.skill} key={index}>
+                              {skill.name}
+                            </p>
+                          ))}
+                        </div>
+                      </div>
+                      <button
+                        className={`${styles.primaryButton} w-2/3`}
+                        onClick={() => redirectToProfile(username)}
+                      >
+                        View
+                      </button>
                     </div>
-                  </div>
-                  <button
-                    className={`${styles.primaryButton} w-2/3`}
-                    onClick={() => redirectToProfile(username)}
-                  >
-                    View
-                  </button>
-                </div>
-              ))}
+                  )
+                )}
           </div>
+          <Pagination
+            pageSize={3}
+            total={freelancers.length}
+            onChange={(page: number, pageSize: number) =>
+              paginatedFreelancers()?.paginate(page)
```

Comment:
same issue as the brief search page

Reply:
made the page a static property for now , pending when the UI is updated